babel.config.js

```diff
@@ -44,7 +44,12 @@ module.exports = {
       },
     ],
   ],
-  ignore: [/@babel[\\|/]runtime/], // Fix a Windows issue.
+  ignore: [
+    // Fix a Windows issue.
+    /@babel[\\|/]runtime/,
+    // Fix const foo = /{{(.+?)}}/gs; crashing.
+    /prettier/,
```

Comment:
It seems to work without, is it still needed?

```suggestion
```

Reply:
Here it crashes if I remove.

![image](https://user-images.githubusercontent.com/42154031/132272411-e00f35ff-9021-47d1-b9e1-796a3638e3c5.png)